models/asset.js

```diff
@@ -0,0 +1,110 @@
+const mongoose = require('../mongoose');
+const uuid = require('uuid');
+const Schema = mongoose.Schema;
+
+const AssetSchema = new Schema({
+
+  id: {
+    type: String,
+    default: uuid.v4,
+    unique: true,
+    index: true
+  },
+  url: {
+    type: String,
+    unique: true,
+    index: true
+  },
+  type: {
+    type: String,
+    default: 'article'
+  },
+  headline: String,
+  summary: String,
+  section: String,
+  subsection: String,
+  authors: [String],
+  publication_date: Date
+},{
+  _id: false,
+  versionKey: false,
+  timestamps: {
+    createdAt: 'created_at',
+    updatedAt: 'updated_at'
+  }
+});
+
+
+/**
+ * Search for assets. Currently only returns all.
+*/
+AssetSchema.statics.search = function(query) {
+
+  return Asset.find(query);
+
+};
+
+/**
+ * Finds an asset by its id.
+ * @param {String} id  identifier of the asset (uuid).
+*/
+AssetSchema.statics.findById = function(id) {
+
+  return Asset.findOne({id});
+
+};
+
+/**
+ * Finds a asset by its url.
+ * @param {String} url  identifier of the asset (uuid).
+*/
+AssetSchema.statics.findByUrl = function(url) {
```

Comment:
This looks good, but we should be clear on what we want to support with findBy functions and what we should handle with queries.